vms/platformvm/service.go

```diff
@@ -830,13 +830,27 @@ func (s *Service) GetCurrentValidators(_ *http.Request, args *GetCurrentValidato
 
 			shares := attr.shares
 			delegationFee := avajson.Float32(100 * float32(shares) / float32(reward.PercentDenominator))
-
-			uptime, err := s.getAPIUptime(currentStaker)
-			if err != nil {
-				return err
+			var uptime *avajson.Float32
+			var connected *bool
+			// Only calculate uptime for primary network validators
+			// TODO: decide whether we want to keep connected for subnet validators
+			// it should be available at this point
```

Comment:
what is the downside of keeping it?

Reply:
we won't be able to show this info. but after removal of connectedsubnet, we won't have this info at all. removed the comment.